R/chat.R

```diff
@@ -14,7 +14,9 @@ NULL
 #'   not supply this directly, but instead set the `OPENAI_API_KEY` environment
 #'   variable.
 #' @param model The model to use for the chat; defaults to GPT-4o mini.
-#' @param quiet If `TRUE` does not print output as its received.
+#' @param echo If `TRUE`, the `chat()` method streams the response to stdout by
```

Comment:
Since it only affects `chat()` it seems weird to make it a class property to me.

Reply:
I thought so too at first, but for interactive uses (the only place where this will be useful) it seemed annoying to have to remember to do `echo=TRUE` every time?

Reply:
But I don't feel strongly--happy to take it out if you prefer